predicthq/endpoints/v1/events/schemas.py

```diff
@@ -61,6 +61,8 @@ class Options:
     state = StringType()
     first_seen = DateTimeType()
     updated = DateTimeType()
+    deleted_reason = StringType()
```

Comment:
Should this be ` StringType(choices=('duplicate', 'cancelled', 'invalid'))

Reply:
The choices are for the search params.